include/cantera/thermo/CoverageDependentSurfPhase.h

```diff
@@ -0,0 +1,476 @@
+                             entropy_coeffs(entropy_coeffs) {}
+        //! index of a target species whose enthalpy and entropy is calculated
+        size_t k;
+        //! index of a species whose coverage affects enthalpy and entropy of
+        //! a target species
+        size_t j;
+        //! array of polynomial coefficients describing coverage-depdendent enthalpy
+        //! [J/kmol] in order of 1st-order, 2nd-order, 3rd-order, and 4th-order
+        //! coefficients
+        vector_fp enthalpy_coeffs;
+        //! array of polynomial coefficients describing coverage-depdendent entropy
+        //! [J/kmol/K] in order of 1st-order, 2nd-order, 3rd-order, and 4th-order
+        //! coefficients
+        vector_fp entropy_coeffs;
+    };
+
+    //! A struct to store sets of parameters used in coverage-dependent enthalpy
+    //! and entropy calculations by a interpolative equation or a piecewise-linear
+    //! equation in CoverageDependentSurfPhase.
+    struct InterpolativeDependency
+    {
+        /*!
+         * @param k index of a target species whose enthalpy and entropy are calculated
+         * @param j index of a species whose coverage affects enthalpy and entropy of
+         *          a target species
+         * @param enthalpy_map map of <coverage[dimensionless], enthalpy[J/kmol]> pairs
+         * @param entropy_map map of <coverage[dimensionless], entropy[J/kmol/K]> pairs
+         */
+        InterpolativeDependency(size_t k, size_t j,
+                                std::map<double, double> enthalpy_map={{0.0, 0.0},
+                                                                       {1.0, 0.0}},
+                                std::map<double, double> entropy_map={{0.0, 0.0},
+                                                                      {1.0, 0.0}}):
+                                k(k), j(j), enthalpy_map(enthalpy_map),
+                                entropy_map(entropy_map) {}
+        //! index of a target species whose enthalpy and entropy are calculated
+        size_t k;
+        //! index of a species whose coverage affects enthalpy and entropy of
+        //! a target species
+        size_t j;
+        //! map of <coverage[dimensionless], enthalpy[J/kmol]> pairs
+        std::map<double, double> enthalpy_map;
+        //! map of <coverage[dimensionless], entropy[J/kmol/K]> pairs
+        std::map<double, double> entropy_map;
+    };
+
+    //! A struct to store sets of parameters used in coverage-dependent heat capacity
+    //! calculations by a log-quadratic equation in CoverageDependentSurfPhase.
+    struct HeatCapacityDependency
+    {
+        /*!
+         * @param k index of a target species whose heat capacity is calculated
+         * @param j index of a species whose coverage affects heat capacity of
+         *          a target species
+         * @param coeff_a coefficient a [J/kmol/K]
+         * @param coeff_b coefficient b [J/kmol/K]
```

Comment:
In the mathematical description, these are c8 and c9, right? I can't find anywhere that "a" and "b" are defined. It might help to provide a little bit more clarity on which of these coefficients is which.